app-emulation/vagrant/vagrant-2.0.2.ebuild

```diff
@@ -19,6 +19,7 @@ LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64"
 IUSE="+virtualbox"
+RESTRICT="test"
```

Comment:
But *why*? Test restriction needs to be documented clearly. Also, I'd like to know that you've actually put some effort into this rather than straight away assuming they can't work.

Reply:
Well I've never tried running the tests before (as far as I remember we never had any tests in vagrant). I thought your report was about that if the tests should not be run then RESTRICT should be in place.

I got some strange error while running the tests:
```
# vagrant-spec test
Execute: vagrant box add box ~/precise64.box
Execute: vagrant up --provider=virtualbox
FExecute: vagrant box add box ~/precise64.box
Test: machine is created successfully
Execute: vagrant up --provider=virtualbox
Execute: vagrant destroy --force
F

Failures:

  1) provider/virtualbox/network/intnet it should behave like provider/network/intnet properly configures an internal network
     Failure/Error:
       def assert_execute(*args, env: nil, &block)
         result = execute(*args, env: env, &block)
         expect(result).to exit_with(0)
         result
       end

     ArgumentError:
       unknown keyword: log
     Shared Example Group: "provider/network/intnet" called from
     # ./test/acceptance/provider-virtualbox/network_intnet_spec.rb:16:in `block (2 levels) in <top (required)>'

  2) provider/virtualbox/linked_clone it should behave like provider/linked_clone creates machine as linked clone
     Failure/Error: DEFAULT_FAILURE_NOTIFIER = lambda { |failure, _opts| raise failure }

       expected command to exit with 0 but got exit code: 1

       stdout: Bringing machine 'default' up with 'virtualbox' provider...
       ==> default: Box 'basic' could not be found. Attempting to find and install...
           default: Box Provider: virtualbox
           default: Box Version: >= 0
       ==> default: Box file was not detected as metadata. Adding it directly...
       ==> default: Adding box 'basic' (v0) for provider: virtualbox
           default: Downloading: basic


       stderr: An error occurred while downloading the remote file. The error
       message, if any, is reproduced below. Please fix this error and try
       again.

       Couldn't open file /tmp/d20180213-24450-1wk7l5f/work/basic
     Shared Example Group: "provider/linked_clone" called from
     # ./test/acceptance/provider-virtualbox/linked_clone_spec.rb:22:in `block (2 levels) in <top (required)>'
```